lbs-dynamodb/DESIGN

```diff
@@ -1,19 +1,21 @@
-LBS-DynamoDB-kv design
-======================
+LBS-DynamoDB design
+===================
 
-The lbs-dynamodb-kv block store is invoked as
+The lbs-dynamodb block store is invoked as
 
-# kivaloo-lbs-dynamodb-kv -s <lbs socket> -t <dynamodb-kv socket>
-      -b <item size> [-1] [-p <pidfile>]
+# kivaloo-lbs-dynamodb -s <lbs socket> -t <dynamodb-kv socket>
+      -m <dynamodb-kv metadata socket> -b <item size> [-1] [-p <pidfile>]
 
 It creates a socket <lbs socket> on which it listens for incoming connections,
 accepts one at a time, and performs I/Os following the LBS protocol.  It
 stores data in Amazon DynamoDB via the kivaloo-dynamodb-kv protocol, using
```

Comment:
Do you still want the `-kv` in `kivaloo-dynamodb-kv`?

Reply:
Yes, since that exposes DynamoDB in a very specific and restrictive way.

Reply:
In `lbs-dynamodb/main.c`, you did:
```
-	fprintf(stderr, "usage: kivaloo-lbs-dynamodb-kv -s <lbs socket>"
￼-	    " -t <dynamodb-kv data socket> -m <dynamodb-kv metadata socket>"
+￼	fprintf(stderr, "usage: kivaloo-lbs-dynamodb -s <lbs socket>"
+￼	    " -t <dynamodb data socket> -m <dynamodb metadata socket>"
```

so I'm guessing that the args for `-t` and `-m` shouldn't be changed?  i.e. they should still be `dynamodb-kv data socket` and `dynamodb-kv metadata socket`?

Reply:
Yes, because those are sockets to `dynamodb-kv` processes.

Reply:
ok, updated.  I cancelled the test runs because there's no point re-running them for such a small change.